solutions/fully-configurable/main.tf

```diff
@@ -0,0 +1,376 @@
+########################################################################################################################
+# Resource Group
+########################################################################################################################
+
+# Input variable validation
+locals {
+  prefix = var.prefix != null ? (var.prefix != "" ? var.prefix : null) : null
```

Comment:
please change this to match the approach in [SCC DA](https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/2f888db2f038d0db6cffbb228cff53fd53652f04/solutions/fully-configurable/main.tf#L30C3-L30C104) so we don't need to use `try` everywhere